style.css

```diff
@@ -118,3 +115,228 @@ ul {
   left: 128px;
   top: 111px;
 }
+
+/* work section */
+.work {
+  display: grid;
+  width: 95%;
+  margin-top: 15%;
+  grid-template-rows: 650px 700px 650px 700px;
+  padding: 50px 10px 10px 10px;
+  background-color: #e5e5e5;
+  row-gap: 100px;
+}
+
+.project {
+  background-color: white;
+  margin-top: 10px;
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100%;
+  padding: 19px;
+  color: white;
+  text-align: center;
+  border-radius: 10px;
+  border: 1px solid #e0e0e0;
+}
+
+.screenshot > img {
+  width: 100%;
+}
+
+.project-name {
+  text-align: left;
+  margin-top: 5px;
+  font-style: normal;
+  font-weight: bolder;
+  font-size: 42px;
+  line-height: 44px;
+  color: #172b4d;
+  text-transform: capitalize;
+}
+
+.meta {
+  margin-top: -33px;
+  display: flex;
+  justify-content: flex-start;
+  font-size: 13px;
+}
+
+.meta > * {
+  padding-right: 10px;
+}
+
+.meta > img {
+  width: 8px;
+  height: 8px;
+  align-self: center;
+  margin-top: -3px;
+}
+
+.social > li > img {
+  height: 25px;
+}
+
+.meta > :first-child {
+  text-transform: uppercase;
+  color: #344563;
+  font-weight: bolder;
+}
+
+.meta > :nth-child(3) {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: capitalize;
+  color: #7a869a;
+}
+
+.meta > :nth-child(5) {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: capitalize;
+  color: #7a869a;
+}
+
+.second-text {
+  margin-top: -3px;
+  color: #344563;
+  text-align: left;
+}
+
+.tools {
+  display: flex;
+  justify-content: space-between;
+  width: 80%;
+  margin: 0;
+  padding: 0;
+  margin-top: -10px;
+}
+
+.tools > li {
+  padding: 0 12px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 8px;
+}
+
+.button {
+  display: block;
+  text-decoration: none;
+  background-color: white;
+  color: #396df2;
+  border: 1px solid #396df2;
+  width: fit-content;
+  padding: 10px 20px;
+  margin-top: 20px;
+  border-radius: 10px;
+  font-size: 20px;
+  font-weight: 500;
+  text-transform: capitalize;
+}
+
+.button:hover {
+  color: white;
+  background-color: #6070ff;
+}
+
+.button:active {
+  background-color: #2230d2;
+  color: white;
+}
+
+.buton:disabled {
+  color: #5e6c84;
+  border-color: #5e6c84;
+}
+
+/* about code */
+.about {
+  background-color: white;
+  display: grid;
+  grid-template-rows: auto auto 70px 70px;
+}
+
+.about > .wrapper,
+.languagues,
+.frameworks,
+.skills {
+  background-color: white;
+}
+
+.about > .wrapper {
+  padding: 50px 20px 20px 20px;
+}
+
+.technologies {
+  margin: 0;
+  padding: 0;
+  background-color: red;
+}
+
+.languagues {
+  display: grid;
+  padding: 0 32px 32px 32px;
+  margin-top: -10px;
+  grid-template-rows: 80px 70px 70px 70px;
+  grid-template-areas:
+    'title'
+    'js'
+    'html'
+    'css';
+  border-bottom: 1px solid #e5e5e5;
+}
+
```

Comment:
- [ ] **OPTIONAL** Following up the above comment about styling. The languages section needs a few modifications as well.
Kindly refer to the given design from the comment above and add the corresponding changes if you got the time.